crates/ibc/src/core/ics04_channel/handler/chan_open_init.rs

```diff
@@ -21,6 +21,10 @@ where
     // An IBC connection running on the local (host) chain should exist.
     let conn_end_on_a = ctx_a.connection_end(&msg.connection_hops_on_a[0])?;
 
+    // let client_id_on_a = conn_end_on_a.client_id();
+    // let client_state_of_b_on_a = ctx_a.client_state(client_id_on_a)?;
+    // client_state_of_b_on_a.confirm_not_frozen()?;
```

Comment:
this should not be commented out

Reply:
[828b40a](https://github.com/cosmos/ibc-rs/pull/531/commits/828b40a5a04d351041b190841fd281e9ac68f467)